src/meta/src/barrier/command.rs

```diff
@@ -141,7 +142,11 @@ pub enum Command {
     /// Barriers from the actors to be dropped will STILL be collected.
     /// After the barrier is collected, it notifies the local stream manager of compute nodes to
     /// drop actors, and then delete the table fragments info from meta store.
-    DropStreamingJobs(Vec<ActorId>),
+    DropStreamingJobs {
+        actors: Vec<ActorId>,
+        unregistered_table_fragment_ids: HashSet<TableId>,
```

Comment:
Did find any places that are using this field `unregistered_table_fragment_ids`. Better to remove it and leave `ReleaseContext` unchanged.

Reply:
It's not used in this PR yet, neither is the `table_id` field of the `NewTableFragmentInfo`. In a later PR they will be used in partial checkpoint management. The state tables belonging to a streaming job will be in a snapshot group. Each snapshot group will have its own committed epoch and safe epoch. The fragment table ids will be used as a identifier to register and unregister snapshot group.